charts/rasa-x/templates/ingress.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+{{- if .Values.ingress.ingressClass.enabled }}
```

Comment:
I think I'd remove the `enabled` field and set `name` to an empty string. If name != "" then use `ingressClassName`, wydt?